dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/ReturnCode.java

```diff
@@ -49,6 +49,8 @@ public enum ReturnCode implements ExitCode {
     ERROR_BAD_JAVA_BITMODE(ExitCode.ERROR_BAD_JAVA_BITMODE), // 31
     // Used by the server scripts when jvm.options has an invalid line.
     ERROR_BAD_JVM_OPTION(32),
+    // Repository connection failed
+    CONNECTION_FAILED(33),
```

Comment:
`InstallException.java` already has CONNECTION_FAILED=33, but `featureUtilityExectuor.java` checks if the return code is defined in `ReturnCode.java`. Adding rc=33 for connection failure. I will need to update featureUtility doc.